### ISSUE TITLE
[BUG] Create new group - Add image from gallery crashing app

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#1458]
**Describe the bug**
I was testing group creation with screen reader. After attempting to add a cover image, the app screen turned black and I was unable to use the back arrow to cancel out of what I was doing. 

**To Reproduce**
1. Go to 'Groups'
2. Click on 'Create Group'
3. Click "Add Cover image"
4. Choose 'Gallery' as image import option
5. Select photo from "Gallery' 

**Expected behavior**
Upon selecting Gallery image, the image would populate the cover image space and user would be taken back to complete the rest of the Group creation form fields.
**Actual behavior**
Upon selecting Gallery image, the screen turns black and user is unable to complete the rest of the Group creation form fields.

**Screenshots**
 
https://user-images.githubusercontent.com/82892873/158239152-a532fa76-a64c-4c52-b51b-c5aa3e55f152.mp4

**Smartphone (please complete the following information):**
Pixel 4a- Android 12

**Additional context**
If the user creates the group first without selecting to add cover image, then goes to edit the cover image of existing group, the screen also goes dark and user is unable to complete and save edits.

**Comment by TodorBachvarov** [comment]
Please test with build v3.3.8 or greater.
